PUBLISHERS.md

```diff
@@ -0,0 +1,66 @@
+# Newspack-Tecnavia Plugin Setup Guide
+
+The Newspack-Tecnavia plugin allows publishers to integrate Tecnavia's e-Edition services into their WordPress site, providing readers with access to digital editions directly from their WordPress website based on authentication configurations.
+
+## Prerequisites
+This Plugin has the following plugins as dependencies:-
+
+1. Newspack.
+2. Woocommerce.
+3. Woocommerce Subscriptions.
```

Comment:
either Woo Subscriptions or Memberships if you want to lock access to Tecnavia bhind a subscription or a membership